src/components/reusable/breadcrumbs/breadcrumbs.ts

```diff
@@ -15,7 +15,7 @@ export class Breadcrumbs extends LitElement {
 
   override render() {
     return html`
-      <nav class="breadcrumbs">
+      <nav aria-label="Breadcrumb" class="breadcrumbs">
```

Comment:
The "Breadcrumbs" string cannot be hard-coded, it must be customizable via prop. Alternatively, application devs can place it directly on the kyn-breadcrumbs tag.

Reply:
Addressed. Moved the aria-label to kyn-breadcrumbs tag.